opimodel/fonts.py

```diff
@@ -12,11 +12,15 @@
 
 class Font(object):
 
-    def __init__(self, fontface, size, style=REGULAR, pixels=True):
+    def __init__(self, name=None, fontface='Liberation Sans',
```

Comment:
Is this the right place to specify the default fontface? Can you just set the 'Default' font name instead? 

Having a default font defined here will potentially conflict with the Default defined in the CSS fonts.def file.